src/structures/Message.js

```diff
@@ -474,7 +474,7 @@ class Message extends Base {
   /**
    * Options that can be passed into {@link Message#edit}.
    * @typedef {Object} MessageEditOptions
-   * @property {string} [content] Content to be edited
+   * @property {string|null} [content] Content to be edited
```

Comment:
```suggestion
   * @property {?string} [content] Content to be edited
```

Reply:
When you want to send an attachment or embed in the edit but want to remove the `content`